readthedocs/redirects/templates/redirects/redirect_form.html

```diff
@@ -76,44 +74,18 @@
     {% endblocktrans %}
   </p>
 
-  {% if redirects|length > 0 %}
-    <h3>{% trans "Redirects" %}</h3>
-    <div class="module">
-      <div class="module-list">
-        <div class="module-list-wrapper">
-          <ul>
-            {% for redirect in redirects %}
-              <li class="module-item">
-                <div>
-                  {{ redirect.get_redirect_type_display }}
-                </div>
-                {% if redirect.get_from_to_url_display %}
-                  <div>
-                    {{ redirect.get_from_to_url_display }}
-                  </div>
-                {% endif %}
-                <ul class="module-item-menu">
-                  <li>
-                    <form method="post" action="{% url "projects_redirects_delete" project.slug %}">
-                      {% csrf_token %}
-                      <input type="hidden" name="id_pk" value="{{ redirect.pk }}" />
-                      <input type="submit" value="{% trans "Remove" %}">
-                    </form>
-                  </li>
-                </ul>
-              </li>
-            {% endfor %}
-          </ul>
-        </div>
-      </div>
-    </div>
-  {% endif %}
-
   <form method="post" action=".">{% csrf_token %}
     {{ form.as_p }}
     <div id="dynamic-redirect" class="empty"></div>
-    <p>
-      <input style="display: inline;" type="submit" value="{% trans "Add" %}">
-    </p>
+    <input type="submit" value="{% trans "Save" %}">
   </form>
+
+  {% if redirect %}
```

Comment:
Is this because we are using the same template for Add than Edit?

Reply:
yeah, on the creation form there is no redirect object, but there is one when editing it.